doc/producers/README.md

```diff
@@ -4,36 +4,77 @@ The 4.x version of the Drupal GraphQL module is built on top of a concept called
 
 A data producer is more or less a function that takes arguments (either from an end user query or static) and resolves these into some other data, for example taking an entity (such as a node) and giving back a particular field.
 
-For example, if we want to make a custom field available for a schema let's imagine we have a `creator` field in the "Article" content type. For this field we can have a producer that takes an entity and returns or resolves the creator field. Let's apply this to our custom schema which alreay defines an "Article" type.
+Lets imagine we want to make a custom field available for a schema, in this case we have a `author` field in the "Article" content type. For this field we can have a producer that takes an entity and returns or resolves the creator field. Let's apply this to our custom schema which alreay defines an "Article" type.
 
 ## Add the field to the schema
 
+In your `.graphqls` file add the schema defintion
+
 ```
-type Article implements NodeInterface {
+type Article {
     id: Int!
-    uid: String
     title: String!
-    render: String
-    creator: String
+    author: String
 }
 ```
 
-We are telling the schema that we have a new field on the "Article" type called creator. This is how a user will consume the API so we are not yet telling how to get that data. For now this would return NULL as we don't yet have a resolver in place.
+We are telling the schema that we have a new field on the "Article" type called "creator". This is how a user will consume the API so we are not yet telling how to get that data. For now this would return NULL as we don't yet have a resolver in place.
```

Comment:
```suggestion
We are telling the schema that we have a new field on the "Article" type called "author". This is how a user will consume the API so we are not yet telling how to get that data. For now this would return NULL as we don't yet have a resolver in place.
```